docs/en/notes/changelog.md

```diff
@@ -1 +1,99 @@
+- he same data transforms in different OpenMMLab 2.x libraries have the same augmentation implementation and the logic of the same arguments, i.e., `Resize` in MMDet 3.x and MMSeg 1.x will resize the image in the exact same manner given the same arguments.
+
+#### Model
+
+- Model:
+- Evaluation
+- Visualization
+
+### Improvements
+
+- Improved training and testing speed of FCOS, RetinaNet, Faster R-CNN, Mask R-CNN, and Cascade R-CNN. The training speed of those models with some common training strategies are also improved, including those with synchronized batch normalization and mixed precision training.
+- Support mixed precision training of all the models. However, some models may got Nan results due to some numerical issues. We will update the documentation and list their results (accuracy of failure) of mixed precision training.
+- Release strong baselines of some popular object detectors. Their accuracy and pre-trained checkpoints will be released.
+
+### Bug Fixes
+
+- DeepFashion dataset: the config and results have been updated.
+
+### New Features
+
+1. Support a general semi-supervised learning framework that works with all the object detectors supported in MMDet 3.x. Please refer to [semi-supervised object detection](../user_guides/semi_det.md) for details.
+2. Enable all the single-stage detectors to serve as region proposal networks. We give [an example of using FCOS as RPN](../user_guides/single_stage_as_rpn.md).
+3. Support a semi-supervised object detection algorithm: [SoftTeacher](https://arxiv.org/abs/2106.09018).
+4. Support [the updated CenterNet](https://arxiv.org/abs/2103.07461).
+5. Support data structures `HorizontalBoxes` and `BaseBoxes` to encapsulate different kinds of bounding boxes. We are migrating to use data structures of boxes to replace the use of pure tensor boxes. This will unify the usages of different kinds of bounding boxes in MMDet 3.x and MMRotate 1.x to simplify the implementation and reduce redundant codes.
+
+### Ongoing changes
+
+1. Test-time augmentation: which is supported in MMDet 2.x, is not implemented in this version due to limited time slot. We will support it in the following releases with a new and simplified design.
+2. Inference interfaces: a unified inference interfaces will be supported in the future to ease the use of released models.
+3. Interfaces of useful tools that can be used in notebook: more useful tools that implemented in the `tools` directory will have their python interfaces so that they can be used through notebook and in downstream libraries.
+4. Documentation: we will add more design docs, tutorials, and migration guidance so that the community can deep dive into our new design, participate the future development, and smoothly migrate downstream libraries to MMDet 3.x.
+5. WiderFace dataset, and Fast R-CNN support: we are verifying their functionalities and will fix related issues soon.
```

Comment:
Is that necessary to add using mmdet to train d2 model here?

Reply:
We can add it in the next version